app/services/rfd.server.ts

```diff
@@ -12,15 +12,25 @@ import type { GetResponseTypeFromEndpointMethod } from '@octokit/types'
 import { asciidoctor, type AdocTypes } from '@oxide/react-asciidoc'
 import { Octokit } from 'octokit'
 
+// hack to get automatic reload on file writes in local mode
+import watchTrigger from '~/.watch-trigger?raw'
 import { generateAuthors, type Author } from '~/components/rfd/RfdPreview'
 import { isTruthy } from '~/utils/isTruthy'
 import { parseRfdNum } from '~/utils/parseRfdNum'
-import { can, Permission } from '~/utils/permission'
+import { can, type Permission } from '~/utils/permission'
 import type { GroupResponse, RfdListResponseItem, RfdResponse } from '~/utils/rfdApi'
 
 import type { Group, User } from './authn.server'
 import { apiRequest } from './rfdApi.server'
 
+const localRepo = process.env.LOCAL_RFD_REPO
+export const isLocalMode = process.env.NODE_ENV === 'development' && localRepo
+
+// sneaky weird thing
+if (isLocalMode) {
+  console.log(watchTrigger)
+}
+
```

Comment:
Seems like we have to actually use the import for it not to get shaken out. I'm sure this can be done in a less silly way than `console.log`.